build.gradle

```diff
@@ -206,7 +206,9 @@ opensearchplugin {
 loggerUsageCheck.enabled = false
 
 // No need to validate pom, as we do not upload to maven/sonatype
-validateNebulaPom.enabled = false
+tasks.matching {it.path in [":validateMavenPom", ":validateNebulaPom", ":validatePluginZipPom"]}.all { task ->
+    task.enabled = false
+}
```

Comment:
Seems like we can keep running the validation by adding the dependency, tested locally works on my machine
```suggestion
tasks.matching {it.path in [":validateMavenPom", ":validateNebulaPom", ":validatePluginZipPom"]}.all { task ->
    task.dependsOn ':generatePomFileForNebulaPublication', ':generatePomFileForPluginZipPublication', ':generatePomFileForMavenPublication'
}
```

Reply:
Updated